### PR TITLE
Set fix localstack container version and set keep-alive header manually / explicit

### PR DESCRIPTION
**Pull request by wem**

### CODE REVIEW

**Comment by aesteve** [comment]
Tests are working locally (but on my laptop they always did).
I'm wondering if this fix would fix the tests on travis, though.

**Comment by wem** [comment]
The problem will happen when you pull the currently latest Localstack image (0.10.3). If you locally already has the 0.10.2 and you don't pull, it will work

**Comment by aesteve** [comment]
Thanks a lot, may I ask you something, please?

Would you mind changing: https://github.com/reactiverse/aws-sdk/blob/master/.travis.yml#L16 

to:

```
      script: ./gradlew --no-daemon -i -Dtests.integration=localstack --stacktrace jacocoTestReport
```

in your branch?

So that we can see if Travis build now passes?

We'll merge your work anyway, and thanks a lot btw. But it'd be really awesome if your changes fix the Travis build!

**Comment by wem** [comment]
Done.
Another question. Are you interested in a Kotlin extension for this library? I think about a collection of AWS SDK extensions with the Await approach, like most of the other Vert.x libraries have.
I ask because it would be some work :)

**Comment by aesteve** [comment]
Ok, I can see on Travis it's still failing unfortunately :( 
At least we gave it a try. Thank you!

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     - stage: test
       name: Build on OpenJDK 8
       jdk: openjdk8
-      script: ./gradlew --no-daemon -i --stacktrace jacocoTestReport
+      script: ./gradlew --no-daemon -i -Dtests.integration=localstack --stacktrace jacocoTestReport
```
Comment (aesteve) [inline .travis.yml]:
```suggestion
      script: ./gradlew --no-daemon -i --stacktrace jacocoTestReport
```

Back to the "non-integration" tests

**Comment by aesteve** [comment]
Regarding your other question, I have to think.
I like working in Kotlin but not sure I have enough knowledge to maintain such a set of extension methods in the long run. Especially with coroutines since I'm not using them that much (mostly using Kotlin + RxJava).

**Comment by aesteve** [comment]
@wem could you please revert the change I requested above, please.

So that we can merge your work :) (I don't have permission to "apply suggestion", unfortunately).

Thank you.

**Comment by wem** [comment]
Hi @aesteve , 

i will do it this evening. Sorry for delay.

**Comment by wem** [comment]
Done

**Comment by aesteve** [comment]
Thanks a lot. 

I'll just take your question out of this PR to open an issue, so that we don't forget about it.
I think Reactiverse maintainers should give their opinion :)

Thanks a lot for your contribution.

**Pull request merged**